cmake/projects/bison/schemes/url_sha1_bison_autotools.cmake.in

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_autotools_project)
+include(hunter_report_broken_package)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+
+hunter_status_debug("Scheme: url_sha1_bison_autotools")
+
+if("@MSVC@")
+  hunter_report_broken_package("Autotools scheme not supported with Visual Studio")
+endif()
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_BUILD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+
+hunter_autotools_project("@HUNTER_EP_NAME@"
+    HUNTER_SELF
+      "@HUNTER_SELF@"
+    URL
+      @HUNTER_PACKAGE_URL@
+    URL_HASH
+      SHA1=@HUNTER_PACKAGE_SHA1@
+    DOWNLOAD_DIR
+      "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
+    SOURCE_DIR
+      "@HUNTER_PACKAGE_SOURCE_DIR@"
+    BUILD_DIR
+      "@HUNTER_PACKAGE_BUILD_DIR@"
+    INSTALL_DIR
+      "@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    GLOBAL_INSTALL_DIR
+      "@HUNTER_INSTALL_PREFIX@"
+    PARALLEL_JOBS
+      "@HUNTER_JOBS_OPTION@"
+    PACKAGE_CONFIGURATION_TYPES
+      "@HUNTER_PACKAGE_CONFIGURATION_TYPES@"
+    EXTRA_FLAGS
+      --disable-rpath
```

Comment:
No need to introduce new scheme just for `EXTRA_FLAGS`. You can add `--disable-rpath` as CMAKE_ARGS, example:
* https://github.com/ruslo/hunter/blob/7799c576559549714362fd59900676a6bf4564b2/cmake/projects/PostgreSQL/hunter.cmake#L36-L40

Reply:
Oh from the documentation I thought any autogen package required required at least one custom scheme.

Reply:
> Oh from the documentation I thought any autogen package required required at least one custom scheme

No, it's not. Feel free to send the patch with update to clarify this.

Reply:
So instead I should copy paste the custom scheme into this file? The content there seems necessary to invoke `./configure` etc.

Reply:
Ignore my question. Will use PostgreSQL as guide here.